main_f4.c

```diff
@@ -349,8 +355,7 @@ board_init(void)
 
 #if INTERFACE_USB
 
-	/* enable Port A GPIO9 to sample VBUS */
-	rcc_peripheral_enable_clock(&RCC_AHB1ENR, RCC_AHB1ENR_IOPAEN);
+	rcc_peripheral_enable_clock(&RCC_AHB1ENR, BOARD_CLOCK_VBUS);
```

Comment:
perhaps this and the change on line 789 should go in a separate commit or be removed entirely

Reply:
Useful and safe, so let's get this in.